### PR TITLE
Aligned the badges to centre

### PR DESCRIPTION
**Pull request by krishnashenoy123**
Let me know if the "Sushiksha-website" has to come at the centre or at the same position

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+<p class="text-center mb-3" align="center">
 <a href="https://github.com/18praneeth/sushiksha-website/issues"><img alt="GitHub issues" src="https://img.shields.io/github/issues/18praneeth/sushiksha-website?style=for-the-badge"></a>
 <a href="https://github.com/18praneeth/sushiksha-website/network"><img alt="GitHub forks" src="https://img.shields.io/github/forks/18praneeth/sushiksha-website?style=for-the-badge"></a>
 <a href="https://github.com/18praneeth/sushiksha-website/stargazers"><img alt="GitHub stars" src="https://img.shields.io/github/stars/18praneeth/sushiksha-website?style=for-the-badge"></a>
+</p>
 
-[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/)
+<p class="text-center mb-3" align="center">
+<a href="https://tutorvita.herokuapp.com/"><img src="https://forthebadge.com/images/badges/made-with-python.svg" border="0" title="Made with Python" /></a>
+</p>
```
Comment (18praneeth) [inline README.md:11]: Tutorvita link 😕 
You can add the sushiksha website link in the anchor tag

**Review by 18praneeth — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (krishnashenoy123) [reply README.md:11]: Yes ill do it.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
-[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/)
+<p class="text-center mb-3" align="center">
+<a href="https://sushiksha.konkanischolarship.com/"><img src="https://forthebadge.com/images/badges/made-with-python.svg" border="0" title="Made with Python" /></a>
+</p>
 
+<p class="text-center mb-3" align="center">
 <a href="http://www.djangoproject.com/"><img src="https://www.djangoproject.com/m/img/badges/djangopowered126x54.gif" border="0" alt="Powered by Django." title="Powered by Django." /></a>
+</p>
 
 #### Project description
```
Comment (18praneeth) [inline README.md:18]: Write a few sentences about Sushiksha, you can refer to the about page of the Website.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,18 @@
 # Sushiksha-website 
 
+<p class="text-center mb-3" align="center">
```
Comment (18praneeth) [inline README.md:3]: At the top add Sushiksha Logo
Make it center aligned

File: README.md [hunk 4]
```diff
@@ -1,12 +1,18 @@
 # Sushiksha-website
```
Comment (18praneeth) [inline README.md:2]: image
Also center align these badges

**Review by 18praneeth — CHANGES_REQUESTED** [review]

**Comment by 18praneeth** [comment]
- [ ] Center align all the badges

- [ ] Create Table of Contents

- [ ] Write about Sushiksha

- [ ] Update the license part in Readme.md 
https://github.com/18praneeth/sushiksha-website#license

**Comment by 18praneeth** [comment]
> * [ ]  Center align all the badges
> * [ ]  Create Table of Contents
> * [ ]  Write about Sushiksha
> * [ ]  Update the license part in Readme.md
>   https://github.com/18praneeth/sushiksha-website#license

Please refer to this task list

File: README.md (hunk 3 above)
Comment (krishnashenoy123) [reply README.md:3]: Where can i get the logo of sushiksha ?

Comment (18praneeth) [reply README.md:3]: https://vkssfalumni.com/wp-content/uploads/2016/11/sushikshalogo-300x300.png

**Comment by krishnashenoy123** [comment]
> > * [ ]   Center align all the badges
> > * [ ]   Create Table of Contents
> > * [ ]   Write about Sushiksha
> > * [ ]   Update the license part in Readme.md
> >   https://github.com/18praneeth/sushiksha-website#license
> 
> Please refer to this task list

Where to add the table of contents ?

**Review by 18praneeth — CHANGES_REQUESTED** [review]
Can you update the License detail

**Comment by 18praneeth** [comment]
> > > * [ ]    Center align all the badges
> > > * [ ]    Create Table of Contents
> > > * [ ]    Write about Sushiksha
> > > * [ ]    Update the license part in Readme.md
> > >   https://github.com/18praneeth/sushiksha-website#license
> > 
> > 
> > Please refer to this task list
> 
> Where to add the table of contents ?

After the Introduction of Sushiksha you can have table of contents like 
- Development
- Installation

**Comment by 18praneeth** [comment]
@krishnashenoy123 Link this License file
https://github.com/18praneeth/sushiksha-website/blob/master/LICENSE

**Comment by krishnashenoy123** [comment]
> * https://github.com/18praneeth/sushiksha-website#license



> > > > * [ ]     Center align all the badges
> > > > * [ ]     Create Table of Contents
> > > > * [ ]     Write about Sushiksha
> > > > * [ ]     Update the license part in Readme.md
> > > >   https://github.com/18praneeth/sushiksha-website#license
> > > 
> > > 
> > > Please refer to this task list
> > 
> > 
> > Where to add the table of contents ?
> 
> After the Introduction of Sushiksha you can have table of contents like
> 
> * Development
> * Installation

I added the Installation, Contributing and License, What should i do for the Development ? should i add it ? i didnt it anywhere in thee readme file.

**Comment by 18praneeth** [comment]
## Development
```git
git checkout develop-<username>
//Create the changes and then commit
git add <filename>
git commit -m "<Message here>"
```
From your `master` branch or your main branch that you are working on, give a PR to our `develop` branch
```!IMPORTANT```: Don't give a PR to the master branch
We only accept PR's to `develop` branch

While sending the PR please mention the issue in the Comment

**Comment by 18praneeth** [comment]
> ## Development
> ```
> git checkout develop-<username>
> //Create the changes and then commit
> git add <filename>
> git commit -m "<Message here>"
> ```
> 
> From your `master` branch or your main branch that you are working on, give a PR to our `develop` branch
> `!IMPORTANT`: Don't give a PR to the master branch
> We only accept PR's to `develop` branch
> 
> While sending the PR please mention the issue in the Comment

Add this to Development @krishnashenoy123

**Review by 18praneeth — APPROVED** [review]

**Pull request merged**